src/api.ts

```diff
@@ -128,8 +128,12 @@ export const cloudRunApiOrigin = () =>
   utils.envOverride("CLOUD_RUN_API_URL", "https://run.googleapis.com");
 export const serviceUsageOrigin = () =>
   utils.envOverride("FIREBASE_SERVICE_USAGE_URL", "https://serviceusage.googleapis.com");
+
 export const apphostingOrigin = () =>
   utils.envOverride("APPHOSTING_URL", "https://firebaseapphosting.googleapis.com");
+export const apphostingP4SAOrigin = () =>
```

Comment:
nit: Why is this called origin? It's not a service origin.

Reply:
Updated this one and another env var to be more accurate